fdbserver/storageserver.actor.cpp

```diff
@@ -8929,6 +8932,19 @@ ACTOR Future<Void> storageServerCore(StorageServer* self, StorageServerInterface
 	self->actors.add(
 	    recurring([&]() { self->transactionTagCounter.startNewInterval(); }, SERVER_KNOBS->TAG_MEASUREMENT_INTERVAL));
 
+	if (SERVER_KNOBS->ENABLE_VERSION_VECTOR) {
+		self->actors.add(recurring(
+		    [&]() {
+			    if (self->logSystem && !self->readyFetchKeys.empty()) {
+				    auto tli = self->logSystem->getBestInterface(self->tag);
+				    if (tli.present()) {
+					    tli.get().unblockPeekRequest.getReply(TLogUnblockPeekRequest());
+				    }
+			    }
+		    },
+		    .1));
```

Comment:
Does this actor wake up every 0.1 seconds? Is it possible to have an AsyncTrigger (or a similar mechanism) that wakes it up when "readyFetchKeys" is populated?